src/main/java/racingcar/RacingCarGameApplication.java

```diff
@@ -12,10 +14,15 @@ public static void main(String[] args) {
         String[] carNames = inputView.insertRacingCarName();
         int totalRacingRound = inputView.insertTotalRacingRound();
 
-        RacingGame racingGame = new RacingGame(carNames);
+        List<String> carNameList = Arrays.asList(carNames);
+        List<RacingCar> racingCars = carNameList.stream()
+                .map(RacingCar::new)
+                .collect(Collectors.toList());
+
+        RacingGame racingGame = new RacingGame(racingCars);
```

Comment:
```suggestion
        RacingGame racingGame = Arrays.stream(carNames)
            .map(RacingCar::new)
            .collect(collectingAndThen(toList(), RacingGame::new))
```

이왕 스트림을 사용하였으니 위처럼 조금 더 간결하게 표현 할 수 있어요.